admyral/compiler/action_parser.py

```
@@ -47,18 +47,26 @@ def parse_action(module_str: str, action_name: str) -> PythonAction:
     }
 
     # extract the action decorator arguments
-    if (
-        len(decorator_list) != 1
-        or (
-            isinstance(decorator_list[0], ast.Call)
-            and decorator_list[0].func.id != "action"
-        )
-        or (
-            isinstance(decorator_list[0], ast.Name) and decorator_list[0].id != "action"
-        )
-    ):
-        raise ValueError("Action function should only have the @action decorator.")
-    action_decorator = decorator_list[0]
+    action_decorator = None
+    for decorator in decorator_list:
+        if isinstance(decorator, ast.Call) and decorator.func.id == "action":
+            action_decorator = decorator
+            break
+        elif isinstance(decorator, ast.Name) and decorator.id == "action":
+            action_decorator = decorator
+            break
```

I think we should actually raise an error here because `display_name` and `display_namespace` must always be defined.